src/Sentry/Protocol/Package.cs

```diff
@@ -37,5 +34,36 @@ public Package(string name, string version)
             Name = name;
             Version = version;
         }
+
+        /// <inheritdoc />
+        public void WriteTo(Utf8JsonWriter writer)
+        {
+            writer.WriteStartObject();
+
+            // Name
+            if (!string.IsNullOrWhiteSpace(Name))
+            {
+                writer.WriteString("name", Name);
+            }
+
+            // Version
+            if (!string.IsNullOrWhiteSpace(Version))
+            {
+                writer.WriteString("version", Version);
+            }
+
+            writer.WriteEndObject();
+        }
+
+        /// <summary>
+        /// Parses from JSON.
+        /// </summary>
+        public static Package FromJson(JsonElement json)
+        {
+            var name = json.GetProperty("name").GetString() ?? "";
```

Comment:
I'm not a fan of these `""` ..

It'll be like: wtf is this empty string and where did it come from.

The type takes the values in the constructor, so a serialized value wtihout them is broken.
I rather we return null here and communicate there's no Package. If that isn't an option the values would be: "unknown" or something clear like that